app/src/renderer/components/wallet/PageSend.vue

```diff
@@ -8,7 +8,7 @@ page(title='Send')
         field-id='send-denomination' field-label='Denomination')
         field#send-denomination(
           type="select"
-          v-model="fields.denom"
+          v-model="fields.denom || denom"
```

Comment:
I am not sure if this works properly. This will set the model to use the property "denom". onSubmit uses "fields.denom". Is it tested, that this works? I proposed another way to set the denom.